include/benchmark/benchmark.h

```diff
@@ -888,7 +932,15 @@ class Benchmark {
   // SetIterationTime(seconds) to report the measured time, which will be used
   // to control how many iterations are run, and in the printing of items/second
   // or MB/second values.
+  // The second overload creates a manual time which measures in some arbitrary 
+  // value set by the user. The Cost Function returns the "cost" of a single 
+  // benchmark iteration's in seconds, to be used to determine the cost
+  // of a single run of the benchmark max iteration estimation and so on.
+  // The unit multiplier changes
   Benchmark* UseManualTime();
+  Benchmark* UseManualTime(
+      std::string units, TimeUnit unit_multiplier = kSecond,
```

Comment:
is there ever a time to call `UseManualTime("myunits")`? Ie, not use the second and third parameters? Consider either removing the default parameters from here or having all the parameters default to sensible ... defaults.

Reply:
I kept the last argument default, but with a better defaulted function name. The other 2 should always be specified though, I think.